provider/src/main/java/com/cloudera/director/google/compute/GoogleComputeInstance.java

```diff
@@ -51,8 +51,6 @@
 
   /**
    * Returns the list of display properties for a Google instance, including inherited properties.
-   *
-   * @return the list of display properties for a Google instance, including inherited properties
```

Comment:
Because of Misha's comment:
https://github.com/cloudera/director-google-plugin/pull/93#discussion_r37189227